.azurePipeline/runDockerComposeTests.sh

```diff
@@ -56,11 +56,14 @@ process_args "$@"
 [[ $TYPE != "integrationtests" && $TYPE != "benchmark" && $TYPE != "tutorials" ]] && echoerr "ERROR: Unrecognized type '$TYPE'. Should be equal to 'integrationtests', 'benchmark' or 'tutorials'" && exit 1
 export TAG=$TAG
 
-commandPrefix="docker-compose -f tools/docker-compose.yml -f tools/ci.yml --project-directory . "
+commandPrefix="docker-compose -f docker-compose.yml -f ci.yml "
 if [[ "$NO_ANSI" == "TRUE" ]]; then
   commandPrefix="$commandPrefix --no-ansi "
 fi
 
+# change current working directory to the tools directory
+cd "${0%/*}/../tools"
```

Comment:
yeah right. I'm sure there is a universe where that command makes intuitive sense...